static/app/views/performance/browser/webVitals/utils/queries/useGoodMehAndBadInteractionsSamplesQuery.tsx

```diff
@@ -0,0 +1,25 @@
+import {useInpSpanSamplesWebVitalsQuery} from 'sentry/views/performance/browser/webVitals/utils/queries/useInpSpanSamplesWebVitalsQuery';
+import type {InteractionSpanSampleRowWithScore} from 'sentry/views/performance/browser/webVitals/utils/types';
+
+type Props = {
+  enabled: boolean;
+  transaction: string;
+};
+
+export function useGoodMehAndBadInteractionsSamplesQuery({transaction, enabled}: Props) {
+  // TODO: Update this function to query good, meh, and bad interactions
+  const {data, isFetching} = useInpSpanSamplesWebVitalsQuery({
+    transaction,
+    enabled,
+    limit: 9,
+  });
+
+  const interactionsTableData: InteractionSpanSampleRowWithScore[] = data.sort(
+    (a, b) => a.inpScore - b.inpScore
+  );
```

Comment:
`useIndexedSpans` has `sorts`, does it not?

Reply:
Yeah it does allow sorting. I just left this in because the function is incomplete and not in use yet. It's supposed to do 3 queries to get a range of good meh and bad samples, then it sorts the results in post. 

Right now it's just querying random samples. I plan to update the rest of this in a follow up

Reply:
here's the follow up PR that completes the page overview stuff https://github.com/getsentry/sentry/pull/66471/files#diff-75e5b41a9a3969e9d21111c2993a9bcbf6e5f2b0c04ac9c8fc2f7cc2a8c92573